.github/workflows/solidity-exporter.yml

```diff
@@ -1,6 +1,10 @@
 name: Export Interfaces And Contracts
 
-on: [push]
+on:
+  push:
+    branches:
+      - main
+      - dev
```

Comment:
Should be pushed also to dev or only to main? IMO it should be pushed only to main

Reply:
@0xDiscotech good q that made me realize something. There should be 2 actions, for publishing a canary and a production release, with different triggers. Updated the PR to reflect that.

Reply:
Nice, this is much better. I will approve this changes, but it would be nice to see that you have to trigger it manually and decide whether the release is a canary or production on the README on a future PR. What do you think?